static/src/js/components/TemplateList/TemplateList.jsx

```diff
@@ -73,7 +124,7 @@ const TemplateList = () => {
       dataKey: 'lastModified',
       title: 'Last Modified',
       className: 'col-auto',
-      dataAccessorFn: (cellData) => cellData.split('T')[0]
+      dataAccessorFn: () => moment().format('MM-DD-YYYY')
```

Comment:
Is this the one we talked about including the time?

Reply:
yes before it was YYYY-MM-DD